### DISCUSSION TITLE
Socks5(h) proxy support of npm cli.

### DISCUSSION THREAD

**Discussion by hongyi-zhao** [npm/feedback#273] [CLI] [answered]
I'm not sure if npm cli supports socks5(h) proxy now. Any hints will be highly appreciated.

Regards,
HY

**Comment by MylesBorins** [comment, marked as answer]
hey @hongyi-zhao a question like this makes a bit more sense to ask over on [the CLI repo issues tracker](https://github.com/npm/cli/issues)

**Reply by hongyi-zhao** [reply]
Thanks a lot for directing me to the correct place.
